diff --git a/riscv-test-suite/env/arch_test.h b/riscv-test-suite/env/arch_test.h
index 3aaf6ae..5d209a1 100644
--- a/riscv-test-suite/env/arch_test.h
+++ b/riscv-test-suite/env/arch_test.h
@@ -1771,6 +1771,9 @@ rvtest_code_begin:
 /**************************************************************************************/
 
 .macro RVTEST_CODE_END          // test is ended, but in no particular mode
+  .global pass
+  pass:
+    nop
   .option push
   .option norvc
   .global rvtest_code_end       // define the label and make it available
